packages/shopify-cart/package.json

```diff
@@ -16,7 +16,7 @@
     "test": "jest --watch",
     "test:ci": "jest --silent --runInBand --collectCoverage",
     "changeset": "changeset",
-    "build": "vite build && tsc && mv dist/src dist/types",
+    "build": "vite build && tsc --project tsconfig.production.json && mv dist/src dist/types",
     "release": "npm run build && changeset publish"
```

Comment:
Can we add a tsc command that uses the dev config? Just as a way to easily type check tests and stuff.

Reply:
Certainly! Added in https://github.com/getnacelle/nacelle-js/pull/258/commits/a9e27456c85109c66b64d8d13b9ec04a0a58f7aa.